modules/tensor_mechanics/src/actions/TensorMechanicsAction.C

```diff
@@ -607,6 +657,61 @@ TensorMechanicsAction::actEigenstrainNames()
              << COLOR_DEFAULT;
 }
 
+void
+TensorMechanicsAction::verifyOrderAndFamilyOutputs()
+{
+  // Ensure material output order and family vectors are same size as generate output
+  std::vector<std::string> components = {"order", "family"};
+  std::string mat = "material_output_";
+  for (auto component : components)
```

Comment:
this is odd

Reply:
I was trying to reduce the length by looping over the order and family, since they use relatively the same structure in errors.

Reply:
get rid of the loop and use the more compact setting of the MooseMultiEnum